en/peripherals/remote_id.md

```diff
@@ -107,6 +129,12 @@ The required baud rate depends on the remote ID used (for Cube ID it must be set
 
 <!-- In theory, a Remote ID (or other MAVLink peripheral) that supports WiFi or wired Ethernet network could also be connected over those links. -->
 
+### DroneCAN Configuration
```

Comment:
@julianoes This is pretty much boilerplate - is it correct for the remote IDs you tried? Anything else to do PX4-side? Do you have to do anything to the id itself to configure it for CAN rather than MAVLINK?